web/Web/web/static/js/document-viewer.js

```diff
@@ -776,6 +776,7 @@ docView.prototype = {
           'ctrl_f_terms_input': $("#search_content").val(),
           'csrfmiddlewaretoken': options.csrfmiddlewaretoken,
           'page_title': document.title,
+          'current_docview_stack_size': parent.viewStack.length,
```

Comment:
The callback on line 762 actually alters the viewStack so the length will be different. Not sure if it's intended for the stack to be changed first before sending the request? Just double checking.

Reply:
If not then store the value in a variable before the callback function call

Reply:
Oh, right. I'd misunderstood what you meant.